automated_qc/tree_cover_loss_qc.py

```diff
@@ -0,0 +1,132 @@
+import os
+import math
+import numpy as np
+import rasterio as rio
+import fiona
+import pandas as pd
+import argparse
+from utils import parse_bounds, concatenate_windows
+from tqdm import tqdm
+from rasterio.windows import from_bounds
+from rasterstats import zonal_stats
+
+# parse arguments
+parser = argparse.ArgumentParser(description='Calculate tree cover loss for admin area')
+parser.add_argument('--admin_area', type=str, help='path to admin area shapefile')
+parser.add_argument('--treecover_threshold', type=int, help='percent threshold for tree cover loss')

```

Comment:
How did you plan to handle contextual layers next time you ran QC? Would the shapefile just already have those layers applied? 

Just to make sure we can exactly reproduce the contextual layer intersections, it might make sense to also pass the shapefiles in for contextual layers, and then use a library like shapely to intersect them all.